composer.json

```diff
@@ -45,7 +45,10 @@
         ],
         "post-create-project-cmd": [
             "php artisan key:generate"
-        ]
+        ],
```

Comment:
we can run "composer test" to run phpunit by version of phpunit what you installed in composer. it doesn't depend on phpunit version what you installed in PC.